indexer/src/infrastructure/repositories/postgres_ethereum_event_repository.rs

```diff
@@ -23,7 +23,7 @@ impl PostgresEthereumEventRepository {
 #[async_trait]
 impl EthereumEventRepository for PostgresEthereumEventRepository {
     async fn list(&self) -> Result<Vec<EthereumEvent>, Box<dyn Error>> {
-        let rows = sqlx::query!(
+        let rows = sqlx::query(
```

Comment:
why change the `query!` macro to the `query` function? AFAIK this removes compiled-time type checking